src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/ProPlanControls.spec.jsx

```diff
@@ -0,0 +1,915 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { graphql, rest } from 'msw'
+import { setupServer } from 'msw/node'
+import { Suspense } from 'react'
+import { MemoryRouter, Route } from 'react-router-dom'
+
+import { TrialStatuses } from 'services/account'
+import { useAddNotification } from 'services/toastNotification'
+import { Plans } from 'shared/utils/billing'
+
+import ProPlanControls from './ProPlanControls'
+
+jest.mock('services/toastNotification')
+jest.mock('@stripe/react-stripe-js')
+jest.mock('./BillingControls', () => () => 'Billing Controls')
+jest.mock('./TotalBanner', () => () => 'Total Banner')
+jest.mock('./UserCount', () => () => 'User Count')
```

Comment:
I would personally remove these mocks just as billing is a super important thing to get right, and making sure everything is fully integrated and works happily together I think is super important.

Reply:
Yup makes sense, I was not fully sure how I felt about it so happy to change it